src/styles/style.css

```diff
@@ -58,7 +58,7 @@
   --light-orange: #f69e45;
 
   /* sizing */
-  --max-footer-height: 69px;
+  --footer-height: 50px;
```

Comment:
I'd suggest we default to units of `rem` so that things scale if people change the text size.
I should have suggested changing it when I added this variable!